src/test/kotlin/com/terraformation/backend/seedbank/search/SearchServiceNestedFieldsTest.kt

```diff
@@ -806,7 +806,6 @@ internal class SearchServiceNestedFieldsTest : SearchServiceTest() {
     // the organization users tables is a special case: a single-value sublist that
     // refers to a child, not a parent. Include it explicitly.
     val usersFieldNames = tables.users.getAllFieldNames("members.user.")
```

Comment:
The blank line here was because the comment doesn't apply to the line after this.